src/test/resources/application.properties

```diff
@@ -1,4 +1,4 @@
-projectVersion=1.0.0
+schemaVersion=1.0.0
```

Comment:
Looks like src properties file still refers to projectVersion

Reply:
updated to schema version